test/augmentation/test_augmentation.py

```diff
@@ -142,3 +142,485 @@ def test_gradcheck(self):
         input = utils.tensor_to_gradcheck_var(input)  # to var
 
         assert gradcheck(RandomHorizontalFlip(p=1.), (input, ), raise_exception=True)
+
+
+class TestColorJitter:
+
+    def smoke_test(self):
+        f = ColorJitter(brightness=0.5, contrast=0.3, saturation=[0.2, 1.2], hue=0.1)
+        repr = "ColorJitter(brightness=0.5, contrast=0.3, saturation=[0.2, 1.2], hue=0.1, return_transform=False)"
+        assert str(f) == repr
+
+    def test_color_jitter(self):
+
+        f = ColorJitter()
+        f1 = ColorJitter(return_transform=True)
+
+        input = torch.rand(3, 5, 5)  # 3 x 5 x 5
+
+        expected = input
+
+        expected_transform = torch.eye(3).unsqueeze(0)  # 3 x 3
+
+        assert_allclose(f(input), expected, atol=1e-4, rtol=1e-5)
```

Comment:
is it noisy ?

Reply:
It makes use of the HSV space. It is noisy as you can check here: https://github.com/kornia/kornia/blob/master/test/color/test_hsv.py#L35